src/components/Arrow/Arrow.tsx

```diff
@@ -21,7 +20,7 @@ interface ArrowPathProps {
 
 const Arrow: FC<ArrowProps> = ({ edge, edgeName, widthOffset, currentNode }) => {
   const styles = useStyles();
-  const isCurrentBranchArrow = isBranchNode(currentNode) && edge.start === currentNode.key;
+  const isCurrentBranchArrow = edge.start === currentNode.key;
```

Comment:
Since we use Branch exclusively with BranchNodes but ActionNodes can also have multiple arrows I think we should rename this variable to just `isCurrentArrow`